net/nginx/Makefile

```diff
@@ -398,6 +400,17 @@ endef
 Package/nginx-ssl/install = $(Package/nginx/install)
 Package/nginx-all-module/install = $(Package/nginx/install)
 
+define Package/nginx/postinst
+#!/bin/sh
+core_number=$$(grep -c ^processor /proc/cpuinfo)
+sed -i "s/^\s*worker_processes\s.*/worker_processes "$$core_number";/" \
+	/etc/nginx/nginx.conf
+endef
```

Comment:
Do we not need a check if we run on the device or on the buildsystem?
If I install nginx with the ImageBuilder to build my own image then */proc/cpuinfo* of my buildsystem is used.

Reply:
Thank you. If we move that script into a uci-defaults it will be called on the device, right?

Reply:
Yes this will run on the device on first boot.

Reply:
So, I will change the PR accordingly :-)

Reply:
I am not the maintainer of this package.
Ping @Ansuel 
I only noticed that while looking through the pullrequest.
But I think so.

Reply:
Nice find